src/methods/join.js

```diff
@@ -458,7 +458,7 @@ export class JoinImplementation {
     let empty_effects = construct_empty_effects(realm);
     let v = realm.intrinsics.empty;
     if (c.consequent instanceof ReturnCompletion) {
-      let negation = AbstractValue.createFromUnaryOp(realm, "!", c.joinCondition)
+      let negation = AbstractValue.createFromUnaryOp(realm, "!", c.joinCondition);
       invariant(negation instanceof AbstractValue);
```

Comment:
It is always a good idea to add a comment to document why you are convinced that the invariant will always be true.